systemtest/src/test/java/io/strimzi/systemtest/security/custom/CustomCaST.java

```diff
@@ -627,17 +449,146 @@ void testClientsCaCertificateRenew(ExtensionContext extensionContext) {
         final Date changedKafkaUserCertStartTime = userCert.getNotBefore();
         final Date changedKafkaUserCertEndTime = userCert.getNotAfter();
 
+        // Print out certificate dates for debug
         LOGGER.info("Initial ClientsCA cert dates: " + initialCertStartTime + " --> " + initialCertEndTime);
         LOGGER.info("Changed ClientsCA cert dates: " + changedCertStartTime + " --> " + changedCertEndTime);
         LOGGER.info("Initial userCert dates: " + initialKafkaUserCertStartTime + " --> " + initialKafkaUserCertEndTime);
         LOGGER.info("Changed userCert dates: " + changedKafkaUserCertStartTime + " --> " + changedKafkaUserCertEndTime);
 
-        assertThat("ClientsCA cert should not have changed.",
-                initialCertEndTime.compareTo(changedCertEndTime) == 0);
-        assertThat("UserCert start date has been renewed",
-                initialKafkaUserCertStartTime.compareTo(changedKafkaUserCertStartTime) < 0);
-        assertThat("UserCert end date has been renewed",
-                initialKafkaUserCertEndTime.compareTo(changedKafkaUserCertEndTime) < 0);
+        // Verify renewal process
+        assertThat("ClientsCA cert should not have changed.", initialCertEndTime.compareTo(changedCertEndTime) == 0);
+        assertThat("UserCert start date has been renewed", initialKafkaUserCertStartTime.compareTo(changedKafkaUserCertStartTime) < 0);
+        assertThat("UserCert end date has been renewed", initialKafkaUserCertEndTime.compareTo(changedKafkaUserCertEndTime) < 0);
+    }
+
+    private void manuallyRenewCa(TestStorage testStorage, SystemTestCertHolder oldCa, SystemTestCertHolder newCa) {
```

Comment:
I think this helper method deserves a proper description with steps (Javadoc)

Reply:
Thanks. Added